### ISSUE TITLE
mv: cannot remove '/data/DoNotStarveTogether/Cluster_1/cluster_token.txt': Device or resource busy

### ISSUE THREAD

**Issue by pointhi** [Jamesits/docker-dst-server#8]
The current script is a bit annoying when someone tries to map the cluster_token into the volume, like:

```yaml
volumes:
- /opt/game_dst/cluster_token.txt:/data/DoNotStarveTogether/Cluster_1/cluster_token.txt:ro
```

This results in the following error message due to the fact ```cluster_token.txt'``` is only mapped into the file directory:
```
mv: cannot remove '/data/DoNotStarveTogether/Cluster_1/cluster_token.txt': Device or resource busy
Error on line 35; RET 1.
```
https://github.com/Jamesits/docker-dst-server/blob/master/entrypoint.sh#L35

Please either do not modify cluster_token.txt, or at least do not modify it when it is already correct.

**Comment by pointhi** [comment]
works like a charm :+1:

**Comment by Jamesits** [comment]
fixed via #9.
